### PR TITLE
Release Notes: 2020-12-17 notes (Operation Center dashboards)

### PR DESCRIPTION
**Pull request by otetard**
image

### CODE REVIEW

**Review by r1chev — APPROVED** [review]

File: docs/releases/2020-12-17.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# 2020-12-17: Operation Center’s Configurable Dashboard System
+
+**SEKOIA.IO’s Operation Center now brings a new dashboard mechanism, that is fully configurable and adaptable to all needs. This new feature is now enabled per default for all SEKOIA.IO customers.**
+
+Dashboard are composed of modular widgets that can be placed where you want. Widgets can be configured to specify the date range, applied filters, display, etc.
```
Comment (gbossert) [inline docs/releases/2020-12-17.md:5]:
```suggestion
Dashboards are composed of modular widgets that can be placed where you want. Widgets can be configured to specify the date range, applied filters, display, etc.
```

File: docs/releases/2020-12-17.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+SEKOIA.IO comes with a pre-configured dashboard that gives a synthetic view of the current community activity, either from a operationnal security perspective (risk level, number of alerts, etc.) and from an activity perspective (list of last posted comments, last created alerts, etc.).
```
Comment (gbossert) [inline docs/releases/2020-12-17.md:7]:
```suggestion
SEKOIA.IO comes with a pre-configured dashboard that gives a synthetic view of the current community activity, either from an operational security perspective (risk level, number of alerts, etc.) or from an activity perspective (list of last posted comments, last created alerts, etc.).
```

File: docs/releases/2020-12-17.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+![Default Dashboard of SEKOIA.IO’s Operation Center](/assets/release-20201217/oc-dashboard-overview.png)
+
+All SEKOIA.IO users are able to create new dashboard that would fit their specific needs. It’s also possible to clone an existing dashboard.
```
Comment (gbossert) [inline docs/releases/2020-12-17.md:11]:
```suggestion
All SEKOIA.IO users are able to create new dashboards that fit their specific needs. It’s also possible to clone an existing dashboard.
```

File: docs/releases/2020-12-17.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Provided Widgets To Monitor SEKOIA.IO’s Operation Center Activity
+
+SEKOIA.IO’s Operation Center provides a several widgets:
```
Comment (gbossert) [inline docs/releases/2020-12-17.md:15]:
```suggestion
SEKOIA.IO’s Operation Center provides several widgets:
```

**Review by gbossert — CHANGES_REQUESTED** [review]

**Comment by otetard** [comment]
Thanks @gbossert for the review and suggestions. I’ve applied you changes and update the date in the notes. I’ve also updated all images to remove all gray borders.

**Review by gbossert — APPROVED** [review]

**Pull request merged**